optimizely/client/client.go

```diff
@@ -54,6 +55,10 @@ func (o *OptimizelyClient) IsFeatureEnabled(featureKey string, userContext entit
 	}()
 
 	projectConfig := o.configManager.GetConfig()
+
+	if reflect.ValueOf(projectConfig).IsNil() {
```

Comment:
So we can't just do a straight up `projectConfig != nil` check because it is an interface?

Reply:
correct